service/resource/persistentvolume/resource.go

```diff
@@ -0,0 +1,128 @@
+package persistentvolume
+
+import (
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/framework"
+	apiv1 "k8s.io/api/core/v1"
+)
+
+const (
+	name                   = "persistentvolume"
+	cleanupAnnotation      = "persistentvolume.giantswarm.io/cleanup"
+	recycleStateAnnotation = "persistentvolume.giantswarm.io/recyclestate"
```

Comment:
What creates those annotations? Doesn't look like it conforms to https://github.com/giantswarm/fmt/blob/master/kubernetes/annotations_and_labels.md#naming

Benefit of conformance is you know immediately what components creates them.